db/migrate/20190219180039_add_comments_to_authentications_table.rb

```diff
@@ -0,0 +1,58 @@
+class AddCommentsToAuthenticationsTable < ActiveRecord::Migration[5.0]
+  def up
+    change_table_comment :authentications, "Authentication information for various resources."
+
+    change_column_comment :authentications, :id, "[builtin] The internal record ID. This is the primary key."
+    change_column_comment :authentications, :name, "Symbolic name for the authentication"
+    change_column_comment :authentications, :authtype, "The authentication type or role, e.g. ssh_keypair, default, amqp, etc."
```

Comment:
> The authentication type or role

Not a comment on the comment ;) But this is my biggest source of confusion.  This is as accurate as I think we can be, it is either the type of authentication (e.g. sometimes it is ssh_keypair or bearer token) and sometimes it is the role (e.g. hawkular, prometheus, metrics).

I would have expected the the type would have been encoded in the STI type and authtype would just be the role.